server/routes.js

```diff
@@ -12,21 +13,35 @@ import webpackConfig from './../webpack.config.babel';
 const urlencodedParser = bodyparser.urlencoded({ extended: true });
 const isProduction = process.env.NODE_ENV === 'production';
 const entryPoint = isProduction ? '/build/' : 'http://localhost:8080/';
+const uploadDir = serverConfig.uploads;
+const password = 'password';
```

Comment:
No, no, not a constant, this should be user configurable. Like suppose, when I upload a file and want it to be encrypted, I'll type in a password and share the URL and password with only one person. Only he / she who knows the password would be able to unzip it.

Reply:
Ok so I should take password from front-end right?

Reply:
Yes.